bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+set -vx
+
+bundle install
+
+# Do any other automated setup that you need to do here
```

Comment:
It is a standard part of the install when you create a new Gem with bundler now.  It is probably more useful for other projects that have scriptable (non-ruby) setup dependencies.  Doesn't hurt to be here though.  Feel free to remove if you don't like it there for some reason.